docs/tutorial/kymotracking.rst

```diff
@@ -501,8 +529,21 @@ This method is slower, since it has to solve some implicit equations, but it doe
      DiffusionEstimate(value=4.432955288469379, std_err=1.565056974828146, num_lags=30, num_points=80, method='gls', unit='um^2 / s'),
      DiffusionEstimate(value=5.378609478528924, std_err=1.7771064499907185, num_lags=30, num_points=80, method='gls', unit='um^2 / s')]
 
+.. note::
+
+    The Generalized Least Squares method for estimating the diffusion constant relies on having a successful positional localization for every frame in a :class:`~lumicks.pylake.kymotracker.kymotrack.KymoTrack`.
+
+    If frames are missing, then this method will throw an error with the suggestion to refine tracks prior to estimation using :ref:`localization_refinement`.
+    If this is not possible, please switch to :ref:`diffusion_cve` or :ref:`diffusion_ols`.
```

Comment:
maybe `raise an exception` rather than `throw an error`? We talk about throwing all the time, but for actual docs I think better to stick with the pythonic terminology